src/algorithms/expected_consensus/_index.md

```diff
@@ -2,119 +2,79 @@
 
 ### Running a leader election
 
-Now, a miner must also check whether they are eligible to mine a block in this round.
-
 Design goals here include:
 
 - Miners should be rewarded proportional to their power in the system
 - The system should be able to tune how many blocks are put out per epoch on expectation (hence "expected consensus").
 
-As discussed in {{<sref election_post>}}, a miner will use the challenge ticket of an ElectionPoSt to uniformly draw a value from 0 to 1 when crafting a block.
-
-The miner gets to draw one such challenge ticket per sector they have committed and must then compare the value derived from the challenge ticket against a target to determine whether they are eligible to mine. This is called finding a winning ticket.
-
-The target is set as follows, for each sector sampled for a ticket from the miner's `ProvingSet` and the number of sectors in the set:
-`target = activePowerInSector/networkPower * EC.ExpectedLeadersPerEpoch * numSectorsMiner / numSectorsSampled`
-
-The target ensures that the miner can express the power across all of their sectors through the tickets they have sampled. Specifically, on expectation, checking `numSectorsSampled` (with `numSectorsSampled = ceil(len(provingSet) * EPoStSampleRate)`) challenge tickets in every epoch, a miner will find `minerPower/networkPower * EC.ExpectedLeadersPerEpoch` winning tickets per epoch on expectation. Note that while the sectorPower may differ based on the challenged sector, i.e. in any given epoch a miner may have an advantage (if picking sectors with more than the average across all their sectors) or a disadvantage (conversely) in their election; but over multiple epochs, on expectation the election will be fair.
-
-```text
-We elaborate on the above claim:
+At a high-level, leader election works as follows:
 
-We want the miner's expected wins over time to be equal to w = minerPower/networkPower * EC.ExpectedLeadersPerEpoch
+- A miner draws an appropriate random seed for this epoch
+- They generate an `ElectionProof` from this seed by using a VRF to generate a signature over the seed at a given epoch, as defined in {{<sref randomness>}}. The `ElectionProof` is the VRF Proof.
+  - If the `ElectionProof`'s normalized digest value (i.e. the VRF Digest) is below a miner-specific `ElectionTarget` determined by the miner's power in SPC, it is valid: the miner can craft a block (see {{<sref block_producer>}}). 
+    - The `ElectionProof` Digest must be normalized according to both the `ExpectedLeadersPerEpoch` in EC (a network parameter) and the maximum value of the digest (based on the hash used to produce it).
+    - The `ElectionTarget` is set as the proportion of the miner's `TotalQualityAdjPower` over the total network quality-adjusted power. Accordingly leader election in EC is proportional to miner power.
```

Comment:
5. (high) Leader election is not using power table lookback `ElectionPowerTableLookback`

This should be the total quality adjusted power at a previous epoch

Reply:
seeing that @sa8 will address this in https://github.com/filecoin-project/specs/pull/967 right?